package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "vue-polar-bear",
-  "version": "0.9.4",
+  "version": "0.9.5",
```

O versionamento do Vue-polar-bear funciona da seguinte maneira:
major.minor.patch
Então no seu caso, ao invés de atualizar por conta de um patch (ajuste), deveria mudar o minor (pequenas melhorias);

```suggestion
  "version": "0.10.4",
```